api/src/main/java/org/openmrs/module/eptsreports/reporting/library/dimensions/DimensionKey.java

```diff
@@ -0,0 +1,10 @@
+package org.openmrs.module.eptsreports.reporting.library.dimensions;
+
+public interface DimensionKey {
+
+  public String getKey();
```

Comment:
What if you would have named it getDimensionKey() and getDimensionValue() just to follow the key pari value a map would contained?

Reply:
If we change to these names, it would give the impression that it is a Map, whereas in reality it is not and does not use a Map in its implementation. 
If those are not clear, maybe you can suggest another names related to the real goal of the methods

Reply:
That is fine, just maintain the way it is